integration-cli/docker_cli_daemon_test.go

```diff
@@ -2238,3 +2238,28 @@ func (s *DockerDaemonSuite) TestDaemonLogOptions(c *check.C) {
 	c.Assert(err, check.IsNil, check.Commentf(out))
 	c.Assert(out, checker.Contains, "{json-file map[]}")
 }
+
+func (s *DockerDaemonSuite) TestDaemonRestartWithAutoRemoveContainer(c *check.C) {
```

Comment:
Unfortunately this test doesn't run on Windows in the current implementation. I'm not sure there's an easy fix though.